### PR TITLE
EOS-19037: Fetch fids using hare command

### PR DESCRIPTION
**Pull request by indrajitzagade**
## Problem Statement
<pre>
  <code>
  Story Ref (if any):
    Fetch fids using hare command
  </code>
</pre>
## Unit testing on RPM done
<pre>
  <code>
  Yes
  </code>
</pre>
## Problem Description
<pre>
  <code>
    Fetch fids using hare command
  </code>
</pre>
## Solution
<pre>
  <code>
    Using hare command for fetching fids instead of a file
  </code>
</pre>
## Unit Test Cases
<pre>
  <code>
    Tested config command by mocking the output of the hare command.
  </code>
</pre>

### CODE REVIEW

File: ha/setup/ha_setup.py [hunk 1]
```diff
@@ -323,6 +324,20 @@ def _update_env(self, node_name: str, node_type: str, cluster_type: str) -> None
         Log.info("CONFIG: Update ha configuration files")
         Conf.save(const.HA_GLOBAL_INDEX)
 
+    def _fetch_fids(self) -> None:
+        """
+        Fetch fids from hare and store in a config file
+        """
+        try:
+            Log.info("Fetch fids from hare and store in a conf file")
+            fids_output, err, rc = self._execute.run_cmd(const.HCTL_FETCH_FIDS, check_error=True)
```
Comment (ajaysrivas) [inline ha/setup/ha_setup.py:333]: we are not planning to store fids in Consul.

Comment (indrajitzagade) [reply ha/setup/ha_setup.py:333]: Not now, but later yes.

**Review by ajay-paratmandali — APPROVED** [review]

**Comment by indrajitzagade** [comment]
@azheregelya Please review this PR.

**Review by azheregelya — APPROVED** [review]

File: ha/setup/ha_setup.py (hunk 1 above)
Comment (indrajitzagade) [reply ha/setup/ha_setup.py:333]: @ajaysrivas I will merge this PR, please let me know if there are further comments, we can handle those separately.

Comment (indrajitzagade) [reply ha/setup/ha_setup.py:333]: Created https://jts.seagate.com/browse/EOS-19248 for storing the FIDs in consul.

**Pull request merged**